docs/pages/components/side-navigation.md

```diff
@@ -20,32 +20,32 @@ The left navigation can always display or expand/collapse using the menu icon wi
     <ul class="fd-side-nav__list">
         <li class="fd-side-nav__item">
             <a class="fd-side-nav__link" href="#">
-                Link Item
+                <span class="fd-side-nav__link-icon--l">Link Item</span>
             </a>
         </li>
         <li class="fd-side-nav__item">
             <a class="fd-side-nav__link is-selected" aira-selected="true" href="#">
-                Link Item
+               <span class="fd-side-nav__link-icon--l">Link Item</span>
             </a>
         </li>
         <li class="fd-side-nav__item">
             <a class="fd-side-nav__link" href="#">
-                Link Item
+                <span class="fd-side-nav__link-icon--l">Link Item</span>
             </a>
         </li>
         <li class="fd-side-nav__item">
             <a class="fd-side-nav__link" href="#">
-                Link Item
+                <span class="fd-side-nav__link-icon--l">Link Item</span>
             </a>
         </li>
         <li class="fd-side-nav__item">
             <a class="fd-side-nav__link" href="#">
-                Link Item
+                <span class="fd-side-nav__link-icon--l">Link Item</span>
```

Comment:
Not sure what all these changes are for.  Many of these do not have icons on them.

Reply:
those classes are for in case people want to add an icon before, and we need these classes to make sure that if others want to add an icon, then the icon is at centered aligned with the first row of text whether one or two text.